fudy/NoteScreen/Component/FramelessWindow.qml

```diff
@@ -111,15 +114,50 @@ ApplicationWindow {
 		onDoubleClicked: function() {
 			root.doubleClicked();
 		}
+
+		onPositionChanged: function(mouse) {
+			if (mouse.x >= parent.width - internal.margin && mouse.y >= parent.height - internal.margin) {
+				mouseArea.cursorShape = Qt.SizeFDiagCursor;
+			} else {
+				mouseArea.cursorShape = Qt.ArrowCursor;
+			}
+		}
 	}
 
-	ColumnLayout {
-		anchors.fill: parent
-		spacing: 0
+	Flickable {
+		id: scrollView
+		width: parent.width
+		height: parent.height - 30
+		contentWidth: parent.width
+		contentHeight: loaderLayout.implicitHeight
+		ScrollBar.vertical: ScrollBar {
+			id: scrollBar
+			hoverEnabled: true
+			active: hovered || pressed
+			orientation: Qt.Vertical
+			anchors {
+				top: parent.top
+				bottom: parent.bottom
+				left: parent.right
+				leftMargin: -20
+			}
+		}
 
-		Loader {
-			id: contentLoader
-			Layout.fillWidth: true
+		ColumnLayout {
+			id: loaderLayout
```

Comment:
Can be removed

Reply:
You mean the id or the Layout?

Reply:
We need this layout to wrap the item in the loader. This will allow the LoaderItem to fill the Flickable and adjust its size when we modify the size of the Note window.

Reply:
Remove id property

Reply:
got it, thank you